### ISSUE TITLE
API To Suspend an App or User

### ISSUE THREAD

**Issue by GrahamC-me** [ServerPilot/API#18]
I had a look through the documentation to see if there's any method of suspending a user or app but nothing jumps out at me.

Did I miss something or is this not currently an option?

Thanks

**Comment by obosor** [comment]
Seems suspension is not available at this moment. Only create and terminate.

**Comment by cheiff** [comment]
Maybe you can remove the domain and create another app with it, to catch its requests.

**Comment by obosor** [comment]
@cheiff Yes, we can do it manually but not from API. If serverpilot team add this function on their API, that will be great.
